README.md

```diff
@@ -7,6 +7,13 @@
 
 The goal of this project is to allow everyone to propose an idea of an opensource, non-profit project and to find a team of developers.
 
+## To run the application you should take the following steps
+
+git clone git@github.com:howtohireme/give-me-poc.git
```

Comment:
please format it properly, using Markdown

like
`git clone git@github.com:howtohireme/give-me-poc.git`
and other snippets in quote block